src/tpc/google-analytics.ts

```diff
@@ -0,0 +1,54 @@
+import { addImports, addTemplate, useNuxt } from '@nuxt/kit'
+import type { Output } from 'third-party-capital'
+import { GooglaAnalyticsData, GoogleAnalytics } from 'third-party-capital'
+import type { RegistryScript } from '../runtime/types'
+import { getTpcScriptContent } from './utils'
+
+export default function googleAnalitycsRegistry() {
+  const nuxt = useNuxt()
+  const { dst } = addTemplate({
+    getContents() {
+      return getTpcScriptContent({
+        data: GooglaAnalyticsData as Output,
+        scriptFunctionName: 'useScriptGoogleAnalytics',
+        use: () => {
+          return { dataLayer: window.dataLayer, gtag: window.gtag }
+        },
+        // allow dataLayer to be accessed on the server
+        stub: ({ fn }) => {
+          return fn === 'dataLayer' ? [] : undefined
+        },
+        tpcKey: 'gtag',
+        tpcTypeImport: 'GoogleAnalyticsApi',
+        augmentWindowTypes: true,
+      })
+    },
+    filename: 'tpc/google-analytics.ts',
```

Comment:
can we namespace to nuxt scripts so it's clear that this module is creating these files